Documentation/microsoft-git.md

```diff
@@ -0,0 +1,39 @@
+Details about the Microsoft/Git Fork
+====================================
+
+The [`microsoft/git` fork of Git](https://github.com/microsoft/git)
+is used to take early performance patches or custom patches for
+[VFS for Git](https://github.com/microsoft/git).
+
+Integration Testing with C# Layer
+---------------------------------
+
+If you have a pull request in `microsoft/git`, then you
+can test integration in VFS for Git using a pull request
+there.
+
+1. Run the [`git - build installers`](https://dev.azure.com/gvfs/ci/_build?definitionId=11)
+   using "Branch/tag" as `refs/pull/<pr-number>/merge`. Leave "package.preview.extension"
+   as `-pr`.
+
+2. Create a topic branch in VFS for Git and modify the
+   [`GVFS/GVFS.Build/GVFS.props`](https://github.com/microsoft/VFSForGit/blob/master/GVFS/GVFS.Build/GVFS.props)
+   file to change the `<GitPackageVersion>` to be the build number of the installer
+   build, followed by the extension (`-pr`). Include "[PR Build]" in your
+   commit message, to make it clear this is not a build based on unmerged commits.
+
+3. Create a pull request in VFS for Git and the functional tests will run. (Make sure
+   the installer build has finished before doing this.) Include "[PR Build]" in your
+   pull request title.
+
+4. Repeat the previous three instructions as you update your `microsoft/git` pull request.
```

Comment:
That is, the Git CI PR builds in step 0 (pushing new commits to your Git topic branch are automatic.  Steps 1..3 are manual.  Don't forget to update the build number each time.